.circleci/config.yml

```diff
@@ -4,11 +4,53 @@ workflows:
   main:
     jobs:
       - test
-
+      - build-binaries-cli
```

Comment:
In this scenario of building two separate binaries, I'd simply this job name to `build-cli`.